.travis.yml

```diff
@@ -55,10 +55,10 @@ script:
   # Check that example compiles with littlefs + spif
   - mbed compile -t GCC_ARM -m K82F -j0
 
-  # Check that example compiles with fatfs + sd
-  - sed -i 's/SPIFBlockDevice bd/SDBlockDevice bd/g' main.cpp
-  - sed -i 's/MBED_CONF_SPIF_DRIVER/MBED_CONF_SD/g' main.cpp
-  - sed -i 's/LittleFileSystem fs/FATFileSystem fs/g' main.cpp
+  # Check that example compiles with fatfs + HeapBlockDevice
+  - sed -i 's/BlockDevice.h/HeapBlockDevice.h/g' main.cpp
+  - sed -i 's/BlockDevice::get_default_instance()/new HeapBlockDevice(2048, 1, 1, 512)/g' main.cpp
```

Comment:
This is a really strange change, but as long as it works.

Would it maybe make more sense to change the configuration such that `HeapBlockDevice` is selected as the default instance, or is this how we'd expect users to specify a HeapBlockDevice?

Reply:
OK, need to explain this one a bit more: If you look at the example main.cpp code, you'll see that the block device we use is the default one for the target, while the file system we use is LittleFS. This changed in earlier PRs, as we thought it should have been the default behavior. Now, we also have a few comments there guiding the user on how to use different block devices and different file systems. The alternative block device used here is `HeapBlockDevice` and the other file system is FAT. Now we could have used other block devices such as SD or SPIF, but it wouldn't work with boards that don't have these components (majority of them), while Heap is something that will work in all of them. All of these are part of the code - the travis config was just adjusted to these changes.